etc/env/example.env

```diff
@@ -1,10 +1,19 @@
 # CanDIGv2 Global Environment Variables
 # - - -
 
+MODE=dev
+
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization jupyter
+CANDIG_MODULES=authz weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization
```

Comment:
Just my opinion, but I find having authz and authorization modules kind of confusing. Not sure what would be the best way to address this but this is something that should be addressed soon-ish